hack/examples/sync.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env bash -x
```

Comment:
```suggestion
#!/usr/bin/env bash 
```

Reply:
any specific reason this should not be enabled by default?

Reply:
https://www.shellcheck.net/wiki/SC2096